wazimap_ng/points/admin/profilecategory_admin.py

```diff
@@ -70,3 +70,9 @@ def get_form(self, request, obj=None, **kwargs):
         form.base_fields["theme"].queryset = qs
 
         return form
+
+    def formfield_for_foreignkey(self, db_field, request, **kwargs):
```

Comment:
the documentation states you should use this function differently. 
why did you opt not to use it that way? 

```
class MyModelAdmin(admin.ModelAdmin):
    def formfield_for_foreignkey(self, db_field, request, **kwargs):
        if db_field.name == "car":
            kwargs["queryset"] = Car.objects.filter(owner=request.user)
        return super().formfield_for_foreignkey(db_field, request, **kwargs)
```

Reply:
Because we already have `formfield_for_foreignkey` in base admin for returning queryset according to permissions.
https://github.com/OpenUpSA/wazimap-ng/blob/staging/wazimap_ng/general/admin/admin_base.py#L83

So super is called first to get the field and after that field's queryset is ordered